docs/dev/cli/cli-architecture.md

```diff
@@ -6,7 +6,7 @@ This document describes the architecture for the CLI component of the TOSCAna So
 ***Note:*** This diagram was created using LucidChart. The source file can be found [here](https://www.lucidchart.com/invitations/accept/6c4ca4c7-d79a-4fee-82ba-6a79e2971f39).
 
 ## ApiController
-The ApiController trys to perform the tasks it receives from the CliController and calls the corresponding methods of the REST API.
+The ApiController tries to perform the tasks it receives from the CliController and calls the corresponding methods of the Retrofit Wrapper.
```

Comment:
Keep it at rest api, maybe mention that you use the Retrofit wrapper to do it though